src/opsinputs/opsinputs_varobswriter_mod.F90

```diff
@@ -963,6 +963,10 @@ subroutine opsinputs_varobswriter_populateobservations( &
       ! TODO(someone): handle this varfield
       ! call Ops_Alloc(Ob % Header % AMSUb_Temp, "AMSUb_Temp", Ob % Header % NumObsLocal, Ob % AMSUb_Temp)
     case (VarField_cloud)
+      call opsinputs_fill_fillelementtype2dfromnormalvariablewithlevels( &
+        Ob % Header % Cloud, "Cloud", Ob % Header % NumObsLocal, Ob % Cloud, &
+        ObsSpace, self % modlevs, "cloudAmount_", "DerivedObsValue", self % GeoVaLsAreTopToBottom, &
+        "cloudAmount_", "DerivedObsError")
```

Comment:
I don't think we should be writing out from DerivedObsError here (as mentioned in https://github.com/JCSDA-internal/ufo/pull/2957). Also can I check why there is a trailing underscore after cloudAmount?

Reply:
OK, thanks James. 
Are you suggesting to replace DerivedObsError with EffectiveError, or simply remove it completely?
The underscore were @ctgh's suggestion to ensure that when the data is on model level, we get cloudAmount_1 to _70 and not cloudAmount1.

Reply:
Ah I see, that makes sense regarding the underscores, thanks for clarifying. 

We would want to replace DerivedObsError with ObsErrorData I think